BankingApp/src/com/company/entities/Admin.java

```diff
@@ -0,0 +1,32 @@
+package com.company.entities;
+
+public class Admin extends User {
+    private String adminName;
```

Comment:
cum e adminName diferit de clientName? e doar un simplu name. Nu ar fi putut fi extras in clasa User?